--- transmission-2.94/libtransmission/transmission.h	2018-05-01 19:21:08.230549000 +0200
+++ transmission-2.94-whithelist/libtransmission/transmission.h	2020-03-15 23:09:47.332511436 +0100
@@ -122,7 +122,7 @@
 
 #define TR_DEFAULT_BIND_ADDRESS_IPV4        "0.0.0.0"
 #define TR_DEFAULT_BIND_ADDRESS_IPV6             "::"
-#define TR_DEFAULT_RPC_WHITELIST          "127.0.0.1"
+#define TR_DEFAULT_RPC_WHITELIST            "*.*.*.*"
 #define TR_DEFAULT_RPC_HOST_WHITELIST              ""
 #define TR_DEFAULT_RPC_PORT_STR                "9091"
 #define TR_DEFAULT_RPC_URL_STR       "/transmission/"

